webapp/src/utils/api.ts

```diff
@@ -1,5 +1,6 @@
 import { paths } from "types/generated/generatedTypes";
 import { constructApiSearchString } from "./helpers";
+import { HTTPExceptionModel } from "types/api";
```

Comment:
If you let VSCode add the import for `HTTPExceptionModel` in alphabetical order, it should place it at the top of the file.
```suggestion
import { HTTPExceptionModel } from "types/api";
import { paths } from "types/generated/generatedTypes";
import { constructApiSearchString } from "./helpers";
```